src/scripts/components/stories/story/blocks/splashscreen/splashscreen.tsx

```diff
@@ -24,23 +32,55 @@ export const SplashScreen: FunctionComponent<StorySectionProps> = ({ ref }) => {
   const imageY = useTransform(scrollYProgress, [0, 1], [0, 150]);
   const opacity = useTransform(scrollYProgress, [0, 0.5, 1], [1, 0.5, 0]);
   const scale = useTransform(scrollYProgress, [0, 1], [1, 1.15]);
+  const dispatch = useDispatch();
+
+  // A story can be "location based" where the splashscreen does not have an image
+  // We consider a location based story if the splashscreen has a location defined
+  const location = story?.splashscreen?.location;
+
+  // We apply the flyTo with a delay. This is a visual effect to make sure the globe container is centered
+  useEffect(() => {
+    if (location) {
```

Comment:
Maybe use `isLocationBased` here. You can then remove the explanatory comment:

`// We consider a location based story if the splashscreen has a location defined`